terra-jupyter-gatk/Dockerfile

```diff
@@ -80,6 +80,7 @@ COPY cnn-models.patch /etc/gatk-$GATK_VERSION/cnn-models.patch
 RUN patch -u /opt/conda/lib/python3.10/site-packages/vqsr_cnn/vqsr_cnn/models.py -i /etc/gatk-$GATK_VERSION/cnn-models.patch
 
 # Older version of nbconvert fail to convert notebooks to html
+RUN pip3 install --upgrade --force-reinstall bleach==6.0.0
```

Comment:
@almahmoud when we try to publish this image the GATK image is failing with `No such file or directory: '/opt/conda/lib/python3.10/site-packages/bleach-6.0.0.dist-info/METADATA'`. Is this addition necessary?

Reply:
This was added by @LizBaldo, not needed for me

Reply:
This error appeared before I added that line @lucymcnatt, the PR as is should not be merged as the GATK image cannot be built. This was my attempts at doing a clean reinstall of the offending package

Reply:
This is the offending line in the dockerfile that originally installs a corrupted version of the bleach package https://github.com/DataBiosphere/terra-docker/blob/c3fe12bb991d1cc8abfdf0ce2c057626d0143016/terra-jupyter-gatk/Dockerfile#L31

I could not locate the requirements.txt file, but my guess is that versions are not pinned in there, which is causing the issue :(

Reply:
thanks Liz! yeah there's a comment above that saying that it does that install to
> Grab the requirements from the python base image and reinstall them. 
> By comparison, simply copying the python sites-packages can result in issues when uninstalling packages: https://github.com/conda/conda/issues/10357

So that's [this file](https://github.com/DataBiosphere/terra-docker/blob/master/terra-jupyter-python/requirements.txt) which doesn't pin most things....

Reply:
I can pin the version there and try again